WalletWasabi/Io/IoManager.cs

```diff
@@ -35,14 +35,16 @@ public void DeleteMe()
 
 		public bool TryReplaceMeWith(string sourcePath)
 		{
-			if (!File.Exists(sourcePath))
+			if (File.Exists(sourcePath))
+			{
+				File.Move(sourcePath, FilePath);
+
+				return true;
+			}
+			else
```

Comment:
Don't need this else statement

Reply:
In the proposed code you need it.
(there is an else statement in the original code as well, simply the 'else' keyword is not written)

Reply:
I'm not really picky about this, so it's fine by me either way.